clients/admin-ui/.env.development

```diff
@@ -0,0 +1 @@
+NEXT_PUBLIC_AUTH_ENDPOINT=http://0.0.0.0:8080/api/v1/oauth/token
```

Comment:
We should follow the same config precedent as the Privacy Centre and put this inside a `config.json` I think

Reply:
Wait a second — @elliotbonneville I think the Admin UI files have been committed to the `main` branch of your fork and it's showing up in this diff now? Probably best to bundle changes into branches on your fork.